### PR TITLE
Modified Uptane Alliance Name

### PR DESCRIPTION
**Pull request by jhdalek55**
changed Uptane Alliance to new legal name

Will be making a few more Linux Foundation-related changes.

Lois

### CODE REVIEW

**Comment by jhdalek55** [comment]
We need to come up with a "shorthand" reference for what we used to call the Alliance. The legal title of the group is now "Joint Development Foundation Projects, LLC, Uptane Series," which is very long and rather awkward to work with. Yet, I agree with some comment Ira made at our last meeting that we should not continue to refer to the group as the "Alliance," because that terminology may not even be our to use anymore. Maybe if we mention the legal name once in a document we could abbreviate it to JDF Project Uptane?

**Comment by JustinCappos** [comment]
> We need to come up with a "shorthand" reference for what we used to call the Alliance. The legal title of the group is now "Joint Development Foundation Projects, LLC, Uptane Series," which is very long and rather awkward to work with. Yet, I agree with some comment Ira made at our last meeting that we should not continue to refer to the group as the "Alliance," because that terminology may not even be our to use anymore. Maybe if we mention the legal name once in a document we could abbreviate it to JDF Project Uptane?

Sounds good to me.

**Comment by jhdalek55** [comment]
Ok. I'm continuing to make small changes. I'll backtrack to include that
abbreviation.

Lois

**Comment by jhdalek55** [comment]
I did another walk through the website to remove mentions of the Alliance and to include the legal title for the new group.I would like Justin to do a pass, but would welcome input from others as well.

File: index.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ implementations available.  Uptane is integrated into [Automotive Grade Linux](h
...
+automobiles, in July 2018 formal standardization of Uptane began under a non-profit consortium called the [Uptane
+Alliance](https://ieee-isto.org/member_programs/uptane-alliance/). [Uptane Standard for Design and Implementation](https://uptane.github.io/uptane-standard/uptane-standard.html) Volume 1.0, which presents procedures for the secure design and implementation of the framework, was released on July 31, 2019 under the auspices of the [IEEE/ISTO Federation](https://ieee-isto.org/). Subsequent versions of the Standard, along with other relevant guidelines for deployment and testing Uptane, will be released through Joint Development Foundation Projects, LLC, Uptane Series, an initiative of the [Linux Foundation](https://www.linuxfoundation.org/). Standardization
```
Comment (JustinCappos) [inline index.md]: Weird line break change

File: overview.md [hunk 2]
```diff
@@ -48,10 +48,9 @@ Engineering became an [associate member](https://www.automotivelinux.org/announc
...
 in part due to the Foundation's adoption of Uptane technology.
 
-On July 2018, the [Uptane Alliance](https://ieee-isto.org/member_programs/uptane-alliance/), 
-a non-profit under [IEEE/ISTO](https://ieee-isto.org/), was formed to
+On July 2018, the [Uptane Alliance](https://ieee-isto.org/member_programs/uptane-alliance/)
+was formed as a non-profit under the umbrella of [IEEE/ISTO](https://ieee-isto.org/) to
 lead the future direction of Uptane research, development, and deployment.
-The Uptane Alliance is currently under the Linux Foundation's Joint Development Foundation.
-The primary task of the Alliance is to be a neutral arbiter that oversees the
+The Uptane Alliance completed its initial task with the release of Volume 1.0 of [Uptane Standard for Design and Implementation](https://uptane.github.io/uptane-standard/uptane-standard.html). The work of the Uptane project, however, continues with the initiation of Joint Development Foundation Projects, LLC, Uptane Series, under the auspices of the [Linux Foundation](https://www.linuxfoundation.org/). It's role, however, remains the same. It serves as a neutral arbiter that oversees the
```
Comment (JustinCappos) [inline overview.md]: Again here.  Can we make this more consistent with the existing text?

File: press.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ css_id: press
 
 # Press
 
+* [The Drive.com-July 23,2019: Top OTA Expert Shows How State Actors Hack into Your Car and What Happens Next: ‘People Will Die’](https://www.thedrive.com/tech/29120/top-ota-expert-shows-how-state-actors-hack-into-your-car-and-what-happens-next-people-will-die?__s=fhb4zzuy2usv5bfnw9cb)
```
Comment (JustinCappos) [inline press.md]: It probably should be in a separate commit.  It's not related here.  

Also, the LWN article is good to cite also

Comment (jhdalek55) [reply press.md]: Ok. I can take that out and add it separately. It makes sense since T=that commit doesn't need to be reviewed.

File: overview.md (hunk 2 above)
Comment (jhdalek55) [reply overview.md]: In terms of the title of the Standards? Not sure what you want changed here.

File: index.md (hunk 1 above)
Comment (jhdalek55) [reply index.md]: fixed

**Comment by jhdalek55** [comment]
I did all but one correction I was unsure of. Please check again and approve so I can either merge, or, if you prefer, request another review.

File: press.md (hunk 3 above)
Comment (jhdalek55) [reply press.md]: DONE

**Comment by jhdalek55** [comment]
I would like these commits merged before I put the link in for the new membership form. Otherwise in doing so, I may create conflicts in the web site. I made the requested changes except for the last one which I did not understand. If you can clarify what you wanted on that last comment, I will change it and this can be merged.
